src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1034,7 +1035,7 @@ private static TextEdit BuildRevertedEdit(TextEdit provisionalTextEdit)
             TextDocument = new VSTextDocumentIdentifier()
             {
                 Uri = delegationDetails.Value.ProjectedUri,
-                ProjectContext = request.ProjectContext
+                ProjectContext = request.Identifier.TextDocumentIdentifier.GetProjectContext()
             },
```

Comment:
Can this be `TextDocument = request.Identifier.TextDocumentIdentifier.WithUri(delegationDetails.Value.ProjectedUri)`?

Reply:
Oh it definitely can, good catch